edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/BookingCancelRequestTO.java

```diff
@@ -8,7 +8,7 @@
 
 public record BookingCancelRequestTO(
   @NotNull
-  @EnumSubset(anyOf = "CANCELLED")
+  @EnumSubset(anyOf = "CANCELLED,REJECTED,DECLINED")
```

Comment:
Please separate this TO.  The `CANCELLED` is for an official TO whereas the other status are only for unofficial endpoints.